code/GrubsConfig.cs

```diff
@@ -111,6 +111,12 @@ public enum TerrainEnvironmentType
 	[ConVar.Replicated( "terrain_environment_type" )]
 	public static TerrainEnvironmentType WorldTerrainEnvironmentType { get; set; } = TerrainEnvironmentType.Sand;
 
+	[ConVar.Replicated( "soundboard_cooldown", Help = "How often (in seconds) players can annoy eachother with sounds." )]
+	public static float SoundboardCooldown { get; set; } = 5;
+
+	[ConVar.Replicated( "soundboard_enabled" )]
+	public static bool SoundboardEnabled { get; set; } = true;

```

Comment:
Might be nice to include a sound volume level that the client can change, from 0 -> 1

Reply:
Great idea!